project-handbook/reuse.md

```diff
@@ -0,0 +1,41 @@
+
+This document helps with the adoption by providing links to relevant material and making recommendations for OpenRail projects.
+
+## Resources for understanding REUSE
+
+Instead of duplicating guides, we refer to helpful material that already exists and is maintained by REUSE:
+
+* [Tutorial](https://reuse.software/tutorial/) on how to become REUSE compliant: make a repository REUSE compliant in simple steps, without delving into details and edge cases.
+* [FAQ](https://reuse.software/faq/) on general questions but also how to deal with edgier cases in your project.
+* [Tools and scripts](https://reuse.software/dev/) specifically for developers, such as the helper tool, API, pre-commit hooks, and CI checks
+
+## Recommended process for OpenRail projects
+
+For Incubation Stage 2, we require projects to be fully REUSE compliant. This is measured by whether the `reuse lint` command exits successfully, but requires that the projects carefully check the licenses and copyright of all their files.
+
+### Analyse your codebase
+
+When thinking about licensing and copyright of all your files, tricky questions may arise:
+
+* Are all files licensed the same way, or have files been copied from other projects under different licenses and copyrights?
+* Who owns the copyright of all the files? Just one organization, or were there contractors or external companies who acted under their own copyright?
+* Are there files that are not be licensed under the "main" license, e.g. icons, fonts, graphics, pictures?
+* Are there trademarks or additional restrictions on parts of the projects, e.g. logos?
+
+### Annotating all files
+
+Once all these questions have been answered, the must be put into practice by adopting the REUSE standard for all your files. Ideally, each file will be annotated directly, so equipped with a comment header containing licensing and copyright information. Only if this isn't possible or not feasible (e.g. for binary or test files), one of the alternatives REUSE offers shall be chosen.
+
+While the required information for REUSE compliance may be put manually into all files, it is strongly advised to make use of e.g. the [REUSE helper tool](https://github.com/fsfe/reuse-tool) that semi-automates many steps. However, manual checks and careful thoughts are mandatory, as described above.
+
+### Ensuring and communicating REUSE adoption
+
+We require two additional steps to ensure that the best practices will be kept:
+
+1. Integrate a REUSE check into your CI, to ensure that new files without proper licensing/copyright annotation issue a warning. Possibilities are explained [here](https://reuse.software/dev/), e.g. via a GitHub action.
+2. Register the project with the [REUSE API](https://api.reuse.software/) and put the resulting badge into your `README` file. This will demonstrate the adoption of this standard and therefore the re-usability to potential users and contributors.
```

Comment:
Nit: The requirement for projects to register to reuse the API is sufficient to demonstrate the adoption of reuse. I don't see why projects should have to display the badge in the README?